maps/away/blueriver/blueriver.dm

```diff
@@ -13,9 +13,12 @@
 		"nav_blueriv_antag"
 	)
 
-/obj/effect/overmap/visitable/sector/arcticplanet/New(nloc, max_x, max_y)
-	name = "[generate_planet_name()], \a [name]"
-	..()
+/obj/effect/overmap/visitable/sector/arcticplanet/Initialize()
+	. = ..()
+	name = "[GLOB.using_map.system_name] [pick("b", "c", "d")], \an [name]"
```

Comment:
It might be better to change `generate_planet_name()` instead. Otherwise, this away site will be oddly named compared to all other exoplanets/moons.

Reply:
This is actual intentional to differentiate it, along with the different rotation, since otherwise it's very easy to confuse it for an exoplanet, even though it's much smaller and entirely different. I had a point about this in the OP:

`(which is no longer (planet name), an arctic planetoid, but instead (system name + letter), an arctic moon - for instance, GSC 452 b. This simulates real-world conventions for exoplanetary bodies!)`

Would you prefer I change it regardless?

Reply:
Since there isn't an actual difference between an exoplanet and a moon (they are all just randomly-named and randomly-placed away sites) in terms of game mechanics, I think it's best to keep it standardized. Otherwise it can easily be meta'd that this is *that* away site.

Reply:
It was previously meta-able by the name of "arctic planetoid" instead of "snow exoplanet". I think we're at a bit of an impasse here, so I'll defer to @Cupax3 here.

Reply:
Quoth cups:
![image](https://user-images.githubusercontent.com/47678781/131233128-5062e45f-1073-43d7-a5c5-b83f523a6ddc.png)